scheduler/src/api/jobs/create_job.rs

```diff
@@ -112,8 +112,9 @@ pub async fn handle_create_job(
         .map_err(|_| internal_server_error("Failed to create scaling sub job"))?;
 
     let sub_jobs = vec![
-        create_sub_job(&state, &job).await?,
-        create_sub_job(&state, &job).await?,
+        scaling_sub_job,
+        create_sub_job(&state, &job, json!({"partial": 80})).await?,
+        create_sub_job(&state, &job, json!({})).await?,
```

Comment:
Just to confirm - these are executed sequentially by the scheduler, right? So It won't start `create_sub_job` until `scaling_sub_job` is completed and it won't start the full one until partial is completed?

Reply:
yes